### ISSUE TITLE
Update psammead-caption appearance

### ISSUE THREAD

**Issue by dr3** [BBC-archive/psammead#270]
**Is your feature request related to a problem? Please describe.**
Based on new UX requirements we need to make captions Reith Sans Italic by default.

**Describe the solution you'd like**
Remove the background colour from the component.

Text:
Regular text should be `Reith Sans Italic`
Text that is marked as italic should be `Reith Sans Regular`
Text that is marked as bold should be `Reith Sans Italic Bold`
Text that is marked as italic AND bold should be `Reith Sans Regular Bold`

All text should have the colour `#757575`, or "Cloud-Dark".

As this creates a visible change, I think this would constitute a "breaking" release. Seeing as we're sub-1.0, this would increment the minor field.

**Testing notes**
We will need examples of each of these font uses on Storybook in order to test them.

What if the font isn't loaded? What about the helvetica fall back? If `Reith Sans Italic` is marked as italic, so the fallback is, will `Reith Sans Italic` be double italic?

What about smaller devices? Any overlap?
What about older devices?

At story refinement, James suggested a regression test for the background colour change, with a specific check to ensure WCAG colour ratio is still satisfactory.

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

**Comment by sareh** [comment]
When this was proposed, we explained that this cannot be the sole way to distinguish captions from body text, since there are scripts that cannot have italic due to it making the text difficult to read.

Several example alternatives were suggested, to separate the caption from the rest of the article text:
- having a coloured border-bottom on the caption block 
- having border-bottom plus left and right borders 
- having a different background colour and text colour for the caption, as to the rest of the article text. 

These were suggestions to be explored by a developer and UX designer pairing on this issue together. This is not an exhaustive list and other alternatives are encouraged to be tried out and explored.

**Comment by sareh** [comment]
Blocked on the investigation ticket work #285

**Comment by bcmn** [comment]
No longer blocked.

**Comment by ChrisBAshton** [comment]
Was this ever resolved? https://github.com/bbc/psammead/issues/270#issuecomment-454463468

I'd hate for us to spend time & effort implementing a caption design that cannot scale for World Service.

**Comment by jimjohnsonrollings** [comment]
For Persian, the plan is that the caption will not be in italic.  Regardless, on all services the font in captions is a different colour from the font in body text.

**Comment by ChrisBAshton** [comment]
Now need to do the work to bring the latest caption into Simorgh.

**Comment by pjlee11** [comment]
Given the requirement that `Text that is marked as italic should be Reith Sans Regular` this is more complicated than it seems at first glance. 

We've put styling of `font-style: italic;` on the caption component in https://github.com/bbc/psammead/pull/365 but are then wanting to override this styling based on a Fragment having the attribute of italic. This feels wrong. I think we should be re-using the existing fragment logic but overriding it to the styling we desire. 

**Example**
Optimo payload in a caption
```
...
  {
    "type": "fragment",
    "model": {
      "text": "Some normal text and",
      "attributes": []
    }
  },
  {
    "type": "fragment",
    "model": {
      "text": "some text marked italic",
      "attributes": ['italic']
    }
  }
...
```

Rather than passing these values directly into Fragment from the Caption container we should reverse the attributes so that `Some normal text and` has the attribute `italic` and `some text marked italic` has no attributes. 

This would save us the hassle of overriding the purpose of italic in Fragment when in the context of a Caption, and in theory should be much simpler logic.

**Comment by ChrisBAshton** [comment]
Hmmm isn't it just something like this?

```scss
[lang=fa] {
  figcaption {
    font-style: italic;
    em {
      font-style: normal;
    }
  }
}
```

I really don't want to mess around with the Optimo payload structure or even have any JS to handle this.

**Comment by pjlee11** [comment]
Ahhh okay, I misunderstood that we wanted to keep the semantics of `<em>` and just override the styling. Yeah, that is simple enough 👍

**Comment by dr3** [comment]
Agreed with @ChrisBAshton, good to maintain semantics

**Comment by ChrisBAshton** [comment]
Have raised #376 to encapsulate the Persian caption work. 👍 We should close this issue (#270) when it's been implemented into Simorgh (tbh we should close it now and have a separate issue for inclusion into Simorgh, but as both are open source we may as well track this one issue across both)

**Comment by ChrisBAshton** [comment]
Actually, raised https://github.com/bbc/simorgh/issues/1457.

Closing.
